_posts/2020-12-10-share-your-knowledge.markdown

```diff
@@ -0,0 +1,59 @@
+
+Here are some examples of things that we have used Knowledge Shares for:
+
+- Sharing early designs for upcoming project work.
+- Going over work-in-progress pull requests and soliciting feedback on the technical approach.
+- Spiking on big-picture projects together, to help us plan what work needs to happen and in which order.
+- Exploring anonymized user sessions to learn more about how our product actually gets used in the real world.
+- Investigating small quality-of-life problems with our codebase's developer experience and working on solutions.
+- Brainstorming about how to address specific pieces of technical debt.
+
+Topics range from the technical to the product-focused, from what has already shipped to what is still on the drawing board. This breadth of focus helps everyone on the team feel like they can contribute. We keep a shared document of evergreen topic ideas and everyone is encouraged to add to it. As Tech Lead, I also send Slack reminders before each meeting to solicit for timely topics.
+
+Throughout the normal course of our week, topics will naturally come up for discussion – topics that would normally need their own meeting to be scheduled. But instead, we can use the time we already have scheduled. For example: if an engineer has feedback on a new feature's design, we don't need to schedule a dedicated meeting for that conversation – we can use a Knowledge Share. It might feel like having recurring "Knowledge Share" meetings only adds more meetings to everyone's already-busy calendars. But in our experience, these Knowledge Shares prevent us from scheduling _yet more_ one-off meetings, so it balances out as a time-saver
+
+One trick I like to pull with Knowledge Shares is to use them to encourage the team's understanding that _learning_ is our paramount goal. That is to say, our goal isn't to _build_ some new feature, but is rather to _learn **how** to build_ that new feature. That's why I emphasize in our Knowledge Shares that _learning is contributing_ (language which I added to our engineering onboarding docs). New team members might not have topics of their own, but they're encouraged to contribute by learning and by asking questions.
+
+When I described the origins of Knowledge Shares above, I said that they worked well on the early Auctions team because the team already had a high level of interpersonal trust. But looking back at those early days, I have to admit that I'm not sure which came first: the Knowledge Shares, or the trust? Having now bootstrapped two product teams at Artsy, I can tell you that Knowledge Shares are a key tool I use as Tech Lead to get teams working well together, quickly. I can't recommend them enough.
```

Comment:
Non-blocking: I think there's an opportunity near the end of this paragraph to really drive the point about building trust. I don't have any suggestions, but if there's anything you could say before "I can't recommend them enough" about how knowledge shares promote trust, it would make this stronger.

(I noticed this/suggested it because I _personally_ reach for sentences like "I can't recommend them enough" when I want to be strong but feel like I don't have anything to back it up. That might not describe you, but if it does, I see you 🤗 )

Reply:
Ah, yes, great point. I'll re-word and link to a blog post I have on the subject of learning as a means to increase team trust: https://ashfurrow.com/blog/building-better-software-by-building-better-teams/